.pull-review

```diff
@@ -0,0 +1,63 @@
+---
+# pull-review config version (required)
+version: 1
+
+# use review requests instead of assignees to assign reviewers to pull requests
+use_review_requests: true
+
+# maximum number of files to evaluate per pull request (set to 0 for no limit)
+max_files: 0 # evaluate all files of the PR
+
+# minimum number of reviewers to assign and notify for a pull request
+min_reviewers: 2
+
+# maximum number of reviewers to assign and notify for a pull request
+max_reviewers: 3
```

Comment:
I would argue that 2 should be enough. I also am not sure if it assigns 2 extra reviewers, or if it is 2 total.

Reply:
Ok, 2 it is then! Pull-review will check how many reviewers are already assigned and take them into account, so that's 2 total, not extra.